### PR TITLE
Add starters into egg pool, roamers

### PR DESCRIPTION
**Pull request by apple096**

### CODE REVIEW

File: src/scripts/GameConstants.ts [hunk 1]
```diff
@@ -710,6 +713,8 @@ namespace GameConstants {
         'Aerodactyl': 'Old Amber',
         'Lileep': 'Root Fossil',
         'Anorith': 'Claw Fossil',
+        'Cranidos': 'Skull Fossil',
+        'Shieldon': 'Armor Fossil',
```
Comment (RedSparr0w) [inline src/scripts/GameConstants.ts:716]: Can we swap these 2 around, just so it's in the same order as `FossilToPokemon`

File: src/scripts/breeding/Breeding.ts [hunk 2]
```diff
@@ -53,6 +56,7 @@ class Breeding implements Feature {
             ['Dratini', 'Dragonair', 'Dragonite'],
             [],
             ['Bagon', 'Shelgon', 'Salamence'],
+            [],
```
Comment (RedSparr0w) [inline src/scripts/breeding/Breeding.ts:61]: Will also need to add empty arrays to the rest of the Eggs that have no Pokemon

Comment (apple096) [reply src/scripts/breeding/Breeding.ts:61]: Done

File: src/scripts/GameConstants.ts (hunk 1 above)
Comment (apple096) [reply src/scripts/GameConstants.ts:716]: Done

File: src/scripts/underground/UndergroundItem.ts [hunk 3]
```diff
@@ -48,8 +48,8 @@ UndergroundItem.addItem('Dome Fossil', 2, [[2,2,2,2,2], [2,2,2,2,2], [2,2,2,2,2]
 UndergroundItem.addItem('Old Amber', 3, [[0,3,3,3], [3,3,3,3], [3,3,3,3], [3,3,3,0]], 0, 'Mine Egg');
 UndergroundItem.addItem('Root Fossil', 4, [[0,0,4,4,4], [0,0,4,4,4], [4,0,0,4,4], [4,4,4,4,4], [0,4,4,4,0]], 0, 'Mine Egg');
 UndergroundItem.addItem('Claw Fossil', 5, [[5,5,5,0,0], [5,5,5,5,0], [0,5,5,5,5], [0,0,0,5,5]], 0, 'Mine Egg');
-// UndergroundItem.addItem('Armor Fossil', 6, [[0,6,6,6,0], [0,6,6,6,0], [6,6,6,6,6], [0,6,6,6,0]], 3);
-// UndergroundItem.addItem('Skull Fossil', 7, [[7,7,7,7], [7,7,7,7], [7,7,7,7], [0,7,7,0]], 3);
+UndergroundItem.addItem('Armor Fossil', 6, [[0,6,6,6,0], [0,6,6,6,0], [6,6,6,6,6], [0,6,6,6,0]], 3);
+UndergroundItem.addItem('Skull Fossil', 7, [[7,7,7,7], [7,7,7,7], [7,7,7,7], [0,7,7,0]], 3);
```
Comment (RedSparr0w) [inline src/scripts/underground/UndergroundItem.ts]: These both need `, 0, 'Mine Egg');`  at the end, instead of the 3 to make it like the other fossils.

Comment (apple096) [reply src/scripts/underground/UndergroundItem.ts]: Done

**Pull request merged**